### PR TITLE
[Feature]: Enabling What's New tab

### LINKED ISSUE #254: Enable what's new in the Devtools extension
Enable whats new tab to display the Edge Devtools VS Code Extension.

### PR DESCRIPTION
**Pull request by mliao95**
This PR enables the What's New tab in the DevTools VS Code extension!

This PR does a handful of things:
- Creates settings flag to enable/disable What's New
- Creates `releaseNote.ts` to handle replacing the content of the What's New page
- Adds `openInNewTab` in `toolsHost.ts` to override the original InspectorFrontendHost function to open links/urls
- Updating drawer logic to account for What's New tab

image

closes #254

### CODE REVIEW

**Comment by codepo8** [comment]
Renaming the titles to keep them shorter: 
"New extension features"
"New in Developer Tools"

**Comment by mliao95** [comment]
With new titles:
image

File: src/host/polyfills/releaseNotes.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export function applyReleaseNotePatch(content: string) {
```
Comment (vidorteg) [inline src/host/polyfills/releaseNotes.ts:4]: can we move this to it's own file? that way the changes are contained to that file and anybody can update the release nots without knowing about the polyfills

**Review by vidorteg — APPROVED** [review]
This one looks amazing. thanks Michael

File: src/host/polyfills/releaseNotes.ts (hunk 1 above)
Comment (mliao95) [reply src/host/polyfills/releaseNotes.ts]: Moved to `releaseNoteContents.ts`

**Pull request merged**